server/server.go

```diff
@@ -64,6 +63,7 @@ func (p *PTMergeServer) Run() {
 		os.Exit(1)
 	}
 	log.Printf("Connected to host FHIR server at %s\n", p.FHIRHost)
+	os.Setenv("FHIRHost", p.FHIRHost)
```

Comment:
I don't think we're using this approach anymore, correct? You can get the current FHIR host from within any `Merger` function, for example:

```go
func (m *Merger) Merge() {
  fhirHost := m.fhirHost
}
```